### DISCUSSION TITLE
[@types/node] crypto: getting error 'Subsequent variable declarations must have the same type' in @types/node@20.11.10

### DISCUSSION THREAD

**Discussion by vmaltarello** [DefinitelyTyped/DefinitelyTyped#68370] [Issues with a @types package] [answered]
I started getting this error on @types/node@20.11.10

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/26143913/8debae24-d041-43cd-8bb4-3ac0fc178dc9)

Info:
TYPESCRIPT: 4.5.4
NODE: 16.17.0
NPM: 8.15.0

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
See #68371

**Reply by jakebailey** [reply]
Now merged; please let me know if it doesn't work.

**Reply by vmaltarello** [reply]
It works, thanks 💯
